final_task/rss_reader/custom_error.py

```diff
@@ -0,0 +1,35 @@
+class Error(Exception):
+    """Base class for other exceptions"""
+
+
+class UrlNotFoundInArgsError(Error):
+    """Raised when can't find url in arguments"""
+    pass
+
+
+class NotEnoughArgumentsError(Error):
+    """Raised when not enough arguments"""
+    pass
+
+
+class NotValidUrlError(Error):
+    """Raised when url is not valid"""
+    pass
+
+
+class ConnectionFailedError(Error):
+    """Raised when can't connect to the url"""
+    message: str
```

Comment:
You don't really need class attribute `message` in this case. Just as it is in the `__init__` will do.